src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -1,12 +1,14 @@
 (ns status-im2.contexts.onboarding.new-to-status.view
-  (:require [quo2.core :as quo]
-            [react-native.core :as rn]
-            [status-im.keycard.recovery :as keycard]
-            [status-im.multiaccounts.recover.core :as multiaccounts.recover]
-            [status-im2.common.resources :as resources]
-            [status-im2.contexts.onboarding.new-to-status.style :as style]
-            [utils.i18n :as i18n]
-            [utils.re-frame :as rf]))
+  (:require
+    [quo2.core :as quo]
+    [react-native.core :as rn]
+    [status-im.keycard.recovery :as keycard]
+    [status-im.multiaccounts.recover.core :as multiaccounts.recover]
+    [status-im2.common.resources :as resources]
+    [status-im2.contexts.onboarding.new-to-status.style :as style]
+    [status-im2.contexts.onboarding.common.navigation-bar.view :refer [navigation-bar]]
```

Comment:
it should be `:as navigation-bar` and then `navigation-bar/view` we're trying to do not use `refer`

Reply:
OK, done!